README.md

```diff
@@ -276,6 +276,17 @@ await GCClient.deleteImage(
 );
 ```
 
+### Download wellness data
```

Comment:
Please use this readme format for new function definitions

https://github.com/Pythe1337N/garmin-connect/blob/master/README.md#getsleepdatadate-string-promisesleepdata

Reply:
reference this post https://github.com/Pythe1337N/garmin-connect/issues/72